localstack/services/events/target.py

```diff
@@ -96,10 +166,35 @@ def _initialize_client(self) -> BaseClient:
         )
         return client
 
+    def _validate_input_transformer(self, input_transformer: InputTransformer):
+        if "InputTemplate" not in input_transformer:
+            raise ValueError("InputTemplate is required for InputTransformer")
+        input_template = input_transformer["InputTemplate"]
+        input_paths_map = input_transformer.get("InputPathsMap", {})
+        placeholders = TRANSFORMER_PLACEHOLDER_PATTERN.findall(input_template)
+        for placeholder in placeholders:
+            if placeholder not in input_paths_map and placeholder not in PREDEFINED_PLACEHOLDERS:
+                raise ValidationException(
+                    f"InputTemplate for target {self.target.get('Id')} contains invalid placeholder {placeholder}."
+                )
+
+    def _get_predefined_template_replacements(self, event: FormattedEvent) -> dict[str, Any]:
+        """Extracts predefined values from the event."""
+        predefined_template_replacements = {}
+        predefined_template_replacements["aws.events.rule-arn"] = self.rule_arn
+        predefined_template_replacements["aws.events.rule-name"] = self.rule_arn.split("/")[-1]
```

Comment:
pars_arn did not work, since it only returns the resource that contains bus and rule name and as you pointed out both bus and rule name can have /. I added `rule_name` as a TargetSender instance variable.
Fixed in: https://github.com/localstack/localstack/pull/10789/commits/1b0366d66320b361982bb3234bfa16cb8bbf730a